config/routes.rb

```diff
@@ -308,16 +306,11 @@
         automate_button_field_changed
         playbook_options_field_changed
         explorer
-        get_ae_tree_edit_key
         group_create
         group_form_field_changed
         group_reorder_field_changed
         group_update
         identify_catalog
-        orchestration_template_add
-        orchestration_template_copy
```

Comment:
I'm not so sure about this one :confused:

Reply:
I was not able to find any usage of those routes neither in url helper methods(`url_for_only_path`,..). I tested orchestration templates forms in UI and it worked.  There is used api in orchestration templates forms  - so maybe thanks to that - there is no need to keep this routes.

or let me know whether I should not remove  `orchestration_template*` routes.

Reply:
It seems it's already using the API, so I'm okay with it.